docs/pages/core-concepts.mdx

```diff
@@ -38,16 +38,22 @@ Read our guides to how [authorization](./architecture/authorization.mdx) and
 
 ### Teleport Proxy Service
 
-The **Teleport Proxy Service** allows for secure access to resources in a your
-infrastructure from the public internet without the need for a VPN. 
+The **Teleport Proxy Service** allows for secure access to resources in your
+infrastructure from the public internet without the need for a VPN.  The
+Proxy Service provides connectivity for users, machines and resources
+within the cluster.
 
-It establishes reverse tunnels to the **Teleport Auth Service** and **Teleport
+It establishes reverse tunnels to the **Teleport Auth Service** for **Teleport
```

Comment:
Would you mind explaining the switch from "and" to "for" here?

Reply:
The connection is for the Teleport Services to connect to the auth service.  The auth service does not perform calls to Teleport services itself, only receives calls directly through its API.  There's no reverse tunnels created for the Teleport Auth service itself.